src/domain/common/memo/memo.service.authorization.ts

```
@@ -0,0 +1,149 @@
+    const privilegeRules: AuthorizationPolicyRulePrivilege[] = [];
+
+    switch (memo.contentUpdatePolicy) {
+      case ContentUpdatePolicy.OWNER:
+        break; // covered via dedicated rule above
+      case ContentUpdatePolicy.ADMINS:
+        const updateContentPrivilegeAdmins =
+          new AuthorizationPolicyRulePrivilege(
+            [AuthorizationPrivilege.UPDATE_CONTENT],
+            AuthorizationPrivilege.UPDATE,
+            POLICY_RULE_MEMO_CONTENT_UPDATE
+          );
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)